Gemfile

```diff
@@ -46,6 +46,7 @@ gem 'simplecov', :require => false, :group => :test
 gem 'react-rails'
 gem 'shippo', git: 'https://github.com/goshippo/shippo-ruby-client'
 gem 'awesome_print'
+gem 'lodash-rails'
```

Comment:
[Lodash](https://lodash.com/docs/4.17.4) is `da bomb` and everyone should be using it.

Reply:
I am `on board` the Lodash train.